### ISSUE TITLE
Design: FE for story and event page using more than one image

### ISSUE THREAD

**Issue by mmiddaugh** [department-of-veterans-affairs/va.gov-cms#16890]
## Issue Description
VAMC editors can add a single image to events and stories. Images are not required but are displayed in the Story list if included. The Secretary has emphasized storytelling, supported by photos and videos and VAMC editors frequently request the ability to add more than one image to stories. 

## Problem(s) to solve 
- How might we enable use of more than one image for VAMC stories while ensuring an accessible and equitable experience for all Veterans?
    - Note: The editorial experience for embedding photos into stories is not ideal and likely needs a comprehensive review and improvement. Does it need to be done before we can enable additional photo use? Related: Enhance editorial experience for embedding photos into stories department-of-veterans-affairs/va.gov-team#13906 
    - Amanda and Danielle ([see comment on #15718](https://app.zenhub.com/workspaces/sitewide-facilities-639f5253e4b702a32376339e/issues/gh/department-of-veterans-affairs/va.gov-cms/15718#issuecomment-1800561775)) recommend a special pilot experiment to test capability for support of one main image and one small supporting image on VAMC Events and Stories only. This pilot would be monitored closely for issues/feedback and could be turned off if problems are discovered. This issue covers the design piece only - separate issues will handle editor guidance, investigation of Drupal for options to address file size compression and support for alt text

## Tasks
- Review selection of VAMC story pages on front end to understand common uses and implications of cropping
- --

- Consider impact/change and update design (as necessary) for (1) individual VAMC stories, (2) story lists, and (3) featured stories considering the following
   - Stories will be limited to one main image and one small supporting image
   - Images are optional 
   - Editors may choose to use one or more than one image for a given story page
   - What is the experience on the story page if if more than one image has been specified? How does the experience compare to a story which specifies only one image?
      -  Example: [Tampa story page](https://www.va.gov/tampa-health-care/stories/veterans-take-quantum-leap-to-recovery/)
   - What is the experience on the story listing page if more than one image has been specified for a given story?
      - Example: [Tampa Story listing page](https://www.va.gov/tampa-health-care/stories/)
   - What is the experience for a featured story if more than one image has been specified? 
      - Example: [Featured story on system home page](https://www.va.gov/tampa-health-care/
---

Consider impact/change and update design (as necessary) for (1) individual VAMC events, (2) event lists, and (3) featured events considering the following
 - Events will be limited to one main image and one small supporting image
   - Images are optional 
   - Editors may choose to use one or more than one image for a given event page
   - What is the experience on the event page if if more than one image has been specified? How does the experience compare to a event which specifies only one image?
      -  Example: [Cincinnati event page](https://www.va.gov/cincinnati-health-care/events/63790/)
   - What is the experience on the event listing page if more than one image has been specified for a given event?
      - Example: [Cincinnati event listing page](https://www.va.gov/cincinnati-health-care/events/)
   - What is the experience for a featured event if more than one image has been specified? 
      - Example: [Featured event on system home page](https://www.va.gov/cincinnati-health-care/)

Note: we'll also need to view and design the editorial experience - this likely needs a separate issue
